src/utils/marketxyz/errors.ts

```diff
@@ -0,0 +1,64 @@
+import { CTokenError, ComptrollerError, MarketSDK } from 'market-sdk';
+
+export async function testForCTokenErrorAndSend(
```

Comment:
is this used anywhere?

Reply:
This is used in `supply`, `withdraw`, `repayBorrow` functions in `utils/marketxyz/index.ts`